### PR TITLE
Get VSTS CI in sync with AppVeyor

### PR DESCRIPTION
**Pull request by TravisEz13**

### CODE REVIEW

**Comment by TravisEz13** [comment]
@anmenaga FYI, This is done.  I'm waiting on VSTS on-board the project to the https://blogs.msdn.microsoft.com/devops/2018/04/27/vsts-public-projects-limited-preview/, but we can merge now.

File: tests/container.tests.ps1 [hunk 1]
```diff
@@ -46,14 +46,14 @@ Describe "Build Linux Containers" -Tags 'Build', 'Linux' {
...
 
-        { Invoke-Docker -Command build -Params @(
+        Invoke-Docker -Command build -Params @(
                 '--pull'
                 '--quiet'
                 '-t'
                 ${Name}
                 $buildArgList
                 $path 
-            ) -SuppressHostOutput} | should -not -throw
```
[the remark this answers is not in the document]
Comment (TravisEz13) [reply tests/container.tests.ps1]: yes, if there is an error, pester just says that there should not have been an error.  Without it, it tells you what the error was and still fails the test as long as the error is terminating.

**Comment by TravisEz13** [comment]
@bergmeister FYI, VSTS CI is enable for this repo.  You mentioned something about VSTS CI in another repo.  Hopefully, I'll have the account setup in the [VSTS Public projects limited preview](https://blogs.msdn.microsoft.com/devops/2018/04/27/vsts-public-projects-limited-preview/) soon, so you can see more than a pass fail.  FYI, VSTS Linux runs in containers (so, I recommend starting with Windows and Mac), but I thought I would point you here in case you wanted to experiment.

**Comment by bergmeister** [comment]
Thanks for letting me know, sounds interesting :)

**Comment by TravisEz13** [comment]
@bergmeister  One other thing you can do in VSTS is have one phase than builds then download the artifact (the build) on multiple phases that test.
https://github.com/Microsoft/vsts-agent/blob/master/docs/preview/yamlgettingstarted-phases.md show phases which depend on a previous phase.  Here is artifact download YAML if you are interested.  

```yaml
steps:
- task: DownloadBuildArtifacts@0
  displayName: Download artifacts
  inputs:
    downloadType: specific
    itemPattern: '**'
  continueOnError: true
```

This could make it efficient to parallelize out the tests to many machines.

**Comment by TravisEz13** [comment]
I'm not saying you have to do this.  I've just been thinking about it and remember you mentioned something.

**Comment by bergmeister** [comment]
Thanks for the link, I am already using YAML for the build system at my workplace. YAML is not available for release definitions (yet) in VSTS but I did not know one can now chain builds like that in VSTS YAML like that (I had to call the VSTS API for chaining last year). I think for PowerShell, having the system tests in a build definition is fine to start with I think but in the future one could think about using release definitions (which have pros and cons with one of the pros being the automatic handling of artifacts)

**Pull request merged**